### PR TITLE
Update Python workflow & related items

### PR DESCRIPTION
**Pull request by josh-hadley**
- created .flake8 config file which stores repo-wide options for flake8 Python linter
- tidied up pythonpackage.yml workflow definition file (update to run in Python3.8, other cleanup)

### CODE REVIEW

File: .flake8 [hunk 1]
```diff
@@ -0,0 +1,10 @@
+[flake8]
+count = True
+show-source = True
+statistics = True
+indent-size = 2
+max-line-length = 128
```
Comment (markusicu) [inline .flake8:6]: I would prefer 100 or so, for more manageable side-by-side diffs, and for working with two files at a time.

**Review by markusicu — COMMENTED** [review]
lgtm, one nit

File: .flake8 (hunk 1 above)
Comment (josh-hadley) [reply .flake8]: 100 SGTM (I had just copied 128 from the workflow config). Done in https://github.com/unicode-org/unicodetools/pull/158/commits/246732e8022269e63cde306c79d7fa957d46f459

**Review by markusicu — APPROVED** [review]
lgtm tnx

please *squash and merge*

**Pull request merged**